web/src/main/webapp/WEB-INF/jsp/team/match/edit.jsp

```diff
@@ -0,0 +1,83 @@
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<my:pagetemplate title="EditMatchForm">
+    <jsp:attribute name="body">
+         <div class="container">
+             <div class="row">
+                 <div class="col-md-8 offset-md-2">
+
+                     <form:form class="form-horizontal match-form" id="match-form-edit" method="post" action="${requestScope['javax.servlet.forward.request_uri']}" modelAttribute="matchRequest" autocomplete="off">
+                         <fieldset>
+                             <legend id="match-form-legend">Edit match</legend>
+
+                             <s:bind path="homeTeamId">
+                                 <div class="form-group row match-form-item ${status.error ? 'has-danger' : ''}">
+                                     <form:label path="homeTeamId" for="hteam" cssClass="col-md-3 col-form-label col-form-label-lg form-control-label">Home Team</form:label>
+                                     <div class="col-md-7">
+                                         <form:select path="homeTeamId" id="hteam" cssClass="match-dropdown-items match-form-team form-control">
+                                             <form:option value="${null}" label="--- Select ---" disabled="true"/>
+                                             <c:forEach items="${teams}" var="team">
+                                                 <form:option value="${team.id}" label="${team.name}" selected="${team.id.equals(matchRequest.homeTeamId) ? 'selected' : ''}" disabled="${!team.id.equals(matchRequest.homeTeamId) ? 'true' : 'false'}"/>
+                                             </c:forEach>
+                                         </form:select>
+                                         <form:errors path="homeTeamId" cssClass="form-control-label" />
+                                     </div>
+                                 </div>
+                             </s:bind>
+
+                             <s:bind path="awayTeamId">
+                                 <div class="form-group row match-form-item ${status.error ? 'has-danger' : ''}">
+                                     <form:label path="awayTeamId" for="ateam" cssClass="col-md-3 col-form-label col-form-label-lg form-control-label">Away Team</form:label>
+                                     <div class="col-md-7">
+                                         <form:select path="awayTeamId" id="ateam" cssClass="match-dropdown-items match-form-team form-control">
+                                             <form:option value="${null}" label="--- Select ---" disabled="true"/>
+                                             <c:forEach items="${teams}" var="team">
+                                                 <form:option value="${team.id}" label="${team.name}" selected="${team.id.equals(matchRequest.awayTeamId) ? 'selected' : ''}" disabled="${!team.id.equals(matchRequest.awayTeamId) ? 'true' : 'false'}"/>
```

Comment:
tohle neni jedina kontrola, že ty data jsou správně, že? ještě se to kontroluje i v modelu?